src/utils/tools/updatingIcons/getEncodedLink.ts

```diff
@@ -0,0 +1,16 @@
+import configs from 'configs'
+
+/**
+ *
+ * Generate encoded links, because gitlab is using an encoded file paths
+ *
+ */
+const getEncodedLink = (pathToFile: string) => {
+  const APILink = configs.GITLAB_READ_API
+  const encodedPath = pathToFile.replace(/\//g, '%2F')
+  const options = 'raw?ref=master'
+  const encodedLink = `${APILink}/${encodedPath}/${options}`
+  return encodedLink
```

Comment:
This used to replace each `/` with `%2F` for gitlab to be able to locate the files